packages/components/src/components/ConfirmationModal/ConfirmationModal.test.tsx

```diff
@@ -0,0 +1,88 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
```

Comment:
Could you please fix the linter here?

Reply:
I would rather like to do it in the whole repo in a separate PR.

Reply:
See https://github.com/Localitos/pluto/pull/1112